src/UIComponents/TrainModel.jsx

```diff
@@ -34,53 +50,94 @@ class TrainModel extends Component {
   };
 
   render() {
+    const { readyToTrainErrors } = this.props;
     return (
       <div>
-        {this.props.selectedFeatures.length > 0 &&
-          this.props.labelColumn &&
-          this.props.selectedTrainer && (
-            <div>
-              <h2>Train the Model</h2>
-              <p>
-                The machine learning algorithm you selected,{" "}
-                {availableTrainers[this.props.selectedTrainer]["name"]}, is
-                going to look for patterns in these features:{" "}
-                {this.props.selectedFeatures.join(", ")} that might help predict
-                the values of the label: {this.props.labelColumn}.
-              </p>
-              <button type="button" onClick={this.onClickTrainModel}>
-                Train model
-              </button>
-              {this.state.showAccuracy && (
+        <h2>Are you ready to train the model?</h2>
+        {readyToTrainErrors.noFeature && (
```

Comment:
I wonder if we could create a JS data structure which maps these errors to their error strings?

Reply:
oooh that's better! I was wondering about something like that I when I realized in the UI I had to know the specific keys to use to access the info in `readyToTrainErrors`.  I'll update.

Reply:
Longer term, this kind of conditional feedback reminds me of the "guide" we built for Oceans at https://github.com/code-dot-org/ml-activities/blob/main/src/oceans/models/guide.js.